doc/userguide/src/CreatingTestData/Variables.rst

```diff
@@ -284,62 +284,70 @@ are imports, setups and teardowns where dictionaries can be used as arguments.
 Accessing list and dictionary items
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-It is possible to access items of lists and dictionaries using special
-syntax `${var}[item]`. Accessing items is an old feature, but prior to
-Robot Framework 3.1 the syntax was `@{var}[item]` with lists and
-`&{var}[item]` with dictionaries. The old syntax was deprecated in
+It is possible to access items of subscriptable variables, e.g. lists and
+dictionaries, using special syntax `${var}[item]`. Accessing items is an old
+feature, but prior to Robot Framework 3.1 the syntax was `@{var}[item]` with
+lists and `&{var}[item]` with dictionaries. The old syntax was deprecated in
 Robot Framework 3.2 and will not be supported in the future.
 
-Accessing list items
-''''''''''''''''''''
+.. _sequence items:
+
+Accessing sequence items
+''''''''''''''''''''''''
 
-It is possible to access a certain item of a list variable with the syntax
-`${var}[index]`, where `index` is the index of the selected value. Indices
-start from zero, negative indices can be used to access items from the end,
-and trying to access an item with too large an index causes an error.
-Indices are automatically converted to integers, and it is also possible to
-use variables as indices. List items accessed in this manner can be used
-similarly as scalar variables.
+It is possible to access a certain item of a `sequence`__ variable (e.g. list,
```

Comment:
Perhaps `a variable containing a sequence` would be a bit more clear than `a sequence variable`, especially when we use the term `list variable` in context where strings aren't allowed.